bin/postman-test/DataDumps/logging/logEntry.js

```diff
@@ -0,0 +1,192 @@
+var dbName = 'logging';
```

Comment:
Please make this a JSON file. All of the other DataDumps files are JSON files, so this one should be as well. In addition, the other DataDumps directories have a populate.sh script that is used for population. Please use the same methodology in this case.

Reply:
In my opinion, JSON file is not maintainable and hard to check syntax error , so I  use mongo js lib .

Reply:
I had add populate.sh and environment file , please check .

Reply:
@weichou1229 Well then you now have a situation where the mechanism to populate data for this one group of tests if different from the others. It needs to be uniform. If you're going to change it for all of them, do that as a separate PR. @cloudxxx8

Reply:
ok , I will change all my PR to JSON file